process/v1_0/transactionProcessorV1_0.go

```diff
@@ -0,0 +1,37 @@
+package v1_0
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-proxy-go/data"
+	"github.com/ElrondNetwork/elrond-proxy-go/process"
+)
+
+type TransactionProcessorV1_0 struct {
```

Comment:
missing comments on exported items

Reply:
removed

Reply:
Some elements without comments remained in this file